addons/website_slides/static/src/xml/channel_management.xml

```diff
@@ -1,12 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <templates>
-    <t t-name="slide.course.join">
-        <div class="p-0 col-md-2">
-            <a role="button"
-                class="btn btn-primary btn-block o_wslides_js_course_join_link text-uppercase font-weight-bold"
-                title="Start Course" aria-label="Start Course Channel"
-                href="#">
-                <span class="cta-title text_small_caps">Join Course</span>
-            </a>
+    <t t-name="slide.quiz.join">
+        <div id ="join_widget" class="alert alert-info d-flex align-items-center justify-content-between">
```

Comment:
Should not use `id` if you can avoid it.
If it's absolutely necessary, please namespace it correctly.